src/main/resources/com/typesafe/sbt/packager/archetypes/scripts/bat-template

```diff
@@ -6,6 +6,7 @@
 @REM Configuration:
 @REM @@APP_ENV_NAME@@_config.txt found in the @@APP_ENV_NAME@@_HOME.
 @setlocal enabledelayedexpansion
+@setlocal enableextensions
```

Comment:
What does this imply? I'm not a windows user and have almost no experience :smile:

Reply:
> I'm not a windows user and have almost no experience 

I also have little experience with BAT scripts - this change required some digging :smile:.

`ENABLEEXTENSIONS` explicitly enables some "newer" (as in, only present in NT-based systems) *command extensions*, like extended `IF` syntax (`IF DEFINED` later in this script). These are almost always enabled anyway, but there are ways to turn them off for compatibilty reasons.

Reply:
Oh wow! Thanks for clarifying this. That makes a lot of sense :smile:

Reply:
Oh wow! Thanks for clarifying this. That makes a lot of sense :smile: